userspace/libsinsp/filterchecks.cpp

```diff
@@ -3517,6 +3517,7 @@ const filtercheck_field_info sinsp_filter_check_gen_event_fields[] =
 	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.source", "Event Source", "the name of the source that produced the event."},
 	{PT_BOOL, EPF_NONE, PF_NA, "evt.is_async", "Async Event", "'true' for asynchronous events, 'false' otherwise."},
 	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.asynctype", "Async-Event Type", "If the event is asynchronous, the type of the event (e.g. 'container')."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.hostname", "Hostname", "The hostname of the underlying host can be customized by setting an environment variable (e.g. FALCO_HOSTNAME for the Falco agent)."},
```

Comment:
not sure if we want to use `SCAP_HOSTNAME` instead of `FALCO_HOSTNAME` here since we are in the libraries, but i also understand that the final use know nothing about `SCAP_HOSTNAME` so it could be fine to use `FALCO_HOSTNAME`

Reply:
:)